### DISCUSSION TITLE
DAI 2023-2024 - Practical work 2 - Jaques Colin & Slimani Walid

### DISCUSSION THREAD

**Discussion by SlWa99** [heig-vd-dai-course/heig-vd-dai-course#155] [Show and tell]
We are going to make a pong game. Each player are represented by a client. The server host the game when 2 player are connected.

**Comment by SlWa99** [comment]
Our repo : https://github.com/CoJaques/Pongly

**Reply by CoJaques** [reply]
And our first release : https://github.com/CoJaques/Pongly/releases/tag/V1.0.0
